.github/workflows/tests.yml

```
@@ -19,12 +19,12 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest']
-        cratedb-version: ['5.2.2']
         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+        cratedb-version: ['5.4.4']
```

As `5.4.5` was declared stable today, can we use it instead?
```suggestion
        cratedb-version: ['5.4.5']
```